soc/nordic/common/uicr/gen_periphconf_entries.py

```diff
@@ -438,7 +478,7 @@ def lookup_tables_get(soc: Soc) -> SocLookupTables:
                 NrfPsel(fun=NrfFun.TPIU_DATA3, port=7, pin=7): Ctrlsel.TND,
             },
         }
-    elif soc == Soc.NRF9280:
+    elif soc == Soc.NRF9280 or (soc == Soc.UNKNOWN and family == Family.SERIES_NRF92X):
```

Comment:
```suggestion
    elif family == Family.SERIES_NRF92X:
```

This seems simpler and more portable.

Reply:
done